lang_qc/db/helper/wells.py

```diff
@@ -208,6 +236,28 @@ def create_for_qc_status(
             wells=wells,
         )
 
+    def create_for_run(self, run_name: str) -> PacBioPagedWells:
+        """
+        Returns `PacBioPagedWells` object that corresponds to the criteria
+        specified by the `page_size` and `page_number` attributes.
+        The `PacBioWell` objects in `wells` attribute of the returned object
+        belong to runs specified by the `run_name` argument and are sorted
+        by the run name and well label.
+        """
+
+        wells = self.get_wells_in_runs([run_name])
+        total_number_of_wells = len(wells)
+        if total_number_of_wells == 0:
+            raise RunNotFoundError(f"Metrics data for run '{run_name}' is not found")
+
+        wells = self.slice_data(wells)
+        return PacBioPagedWells(
+            page_number=self.page_number,
```

Comment:
I think it would be nice to unplug these pagination parameters too, but not today